llvm/docs/CommandGuide/llvm-symbolizer.rst

```diff
@@ -207,6 +207,14 @@ Example 7 - Addresses as symbol names:
   foz
   /tmp/test.h:1:0
 
+Example 8 - Addresses having approximate line info:
```

Comment:
update "approximate line info" as --skip-line-zero is used now?

Reply:
I think the description below at line number 229 states that '--skip-line-zero' is an option which shows approximate line number for those address having no line number. Changing it here dosen't make sense to me.  I can change this to `approximate line number`. @bd1976bris , any comments?

Reply:
I suggest: `--skip-line-zero output for a line zero address:`